labs/ipfs/reference-content/install-ipfs-desktop.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: Using Scaleway IPFS Pinning with IPFS Desktop
+  description: This page shows you how use Scaleway IPFS Pinning with IPFS Desktop
+content:
+  h1: Using Scaleway IPFS Pinning with IPFS Desktop
+  paragraph: This page shows you how to use Scaleway IPFS Pinning with IPFS Desktop
+tags: ipfs pinning storage ipfs-storage volume desktop labs web3
+dates:
+  validation: 2023-09-26
+  posted: 2023-09-26
+categories:
+  - labs
+---
+
+With Scaleway IPFS pinning, you can permanently store your data on the public IPFS network. You can either [create](/labs/ipfs/api-cli/volumes-operations) and manage your volume  via the [Kubo client](https://docs.ipfs.tech/install/command-line/) or through [IPFS Desktop](https://docs.ipfs.tech/install/ipfs-desktop/). 
+
+This article shows you how to add Scaleway as your remote pinning service using IPFS Desktop.
```

Comment:
I think it's better as reference content or even potentially a how to, because from what I've understood you HAVE to install and configure IPFS desktop to be able to use the product. Therefore it makes sense to me that this documentation should be integrated into the core product doc available on the side menu, whereas tutorials are more of an "extra thing you can do with this product".